pcp/PCPDynamicMeter.c

```diff
@@ -139,12 +140,9 @@ static bool PCPDynamicMeter_validateMeterName(char* key, const char* path, unsig
 }
 
 // Ensure a meter name has not been defined previously
-static bool PCPDynamicMeter_uniqueName(char* key, const char* path, unsigned int line, PCPDynamicMeters* meters) {
+static bool PCPDynamicMeter_uniqueName(char* key, PCPDynamicMeters* meters) {
    if (DynamicMeter_search(meters->table, key, NULL) == false)
       return true;
```

Comment:
```suggestion
   return !DynamicMeter_search(meters->table, key, NULL);
```

Remove the `return false;` accordingly.